bpf/lib/eps.h

```diff
@@ -46,27 +46,63 @@ lookup_ip4_endpoint(struct iphdr *ip4)
 }
 
 #if defined POLICY_EGRESS && defined LXC_ID
+/* IPCACHE_STATIC_PREFIX gets sizeof non-IP, non-prefix part of ipcache_key */
+#define IPCACHE_STATIC_PREFIX							\
+	(8 * (sizeof(struct ipcache_key) - sizeof(struct bpf_lpm_trie_key)	\
```

Comment:
Out of curiously , Do we need to worry about these `IPCACHE_STATIC_PREFIX ` value varying due to structs `ipcache_key` and `bpf_lpm_trie_key ` being packed or padded differently on different systems? 
Or does BPF compiler takes care of the struct padding to be consistent?

Reply:
Interesting point, I think that if we want to be careful then we should probably add the packed attributes to the structure definitions for these. /cc @borkmann

Reply:
I packed the `struct ipcache_key`.

Reply:
@manalibhutiyani you mean different archs? We're using `-target bpf`, so so it's 64bit everywhere no matter the underlying arch. See also the difference compared to invoking clang with native/default target (which we're not doing in Cilium).  https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/bpf/bpf_devel_QA.txt#n519